packages/modules/cra-template-web-viewer/template/src/Header.tsx

```diff
@@ -26,6 +26,7 @@ export const Header: React.FC<HeaderProps> = ({
           className={styles.button}
           onClick={handleLogin}
           styleType={"cta"}
+          size={"small"}
```

Comment:
I thought we were removing the header in the cra template for this release and just signing in on startup?

Reply:
I wasn't aware of that, I wouldn't be opposed to it tho. Should that go in a separate pr then? I snuck the style changes in here cause I thought they were miniscule

Reply:
yes I'll add a PBI for it. I think it was actually removed with the changes to the template for sandbox export that we ended up reverting